tests/conftest.py

```diff
@@ -45,7 +47,6 @@ def fixture_vcr_config() -> dict[str, Any]:
             ANTHROPIC_API_KEY_HEADER,
             "cookie",
         ],
-        "match_on": ["method", "host", "path", "query"],
```

Comment:
@mskarlin do you know why we overrode the default `match_on` here? The default has 6 entries, why did we down-specify to 4?